fe/fe-core/src/test/java/org/apache/doris/nereids/rules/rewrite/logical/ColumnPruningTest.java

```diff
@@ -177,6 +178,20 @@ public void testPruneColumns4() {
                 );
     }
 
+    @Test
+    public void pruneCountStarStmt() {
```

Comment:
please add more test such as
```sql
SELECT COUNT(*), SUM(1) + SUM(2) FROM t GROUP BY a, b;
SELECT COUNT(*) + 1, COUNT(c) FROM t GROUP BY a, b;
```

Reply:
To support such expressions like `COUNT(*) + 1`, we need to add project above the agg node, I remeber that another contributor is doing this job.